### DISCUSSION TITLE
WebP Doubts

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#5423] [Help]
What is the default compression will be used  by IM while resizing the WebP Image ?

**Comment by snibgo** [comment]
As far as I know, a default is not documented. So you should not assume any particular default.

**Reply by programmerarunkumar** [reply]
so I have to define either Lossy or LossLess while resizing the webp image ?

**Comment by snibgo** [comment]
Webp options are documented at https://imagemagick.org/script/webp.php . Some options, but not all, have documented defaults. If you want an option to have a particular value that is not a documented default, you should specify it.
